providers/linux/testdata/ubuntu2204/etc/os-release

```diff
@@ -0,0 +1 @@
+../usr/lib/os-release
```

Comment:
Confirm that this is here to exercise symlink following?

Reply:
That's correct. I'm trying to match the layout from the OS.